packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -460,7 +460,9 @@ export default class CFDStore extends BaseStore {
             zipCode: address_postcode,
             ...(this.account_type.type === 'all' ? { sub_account_category: 'swap_free' } : {}),
             ...(values.server ? { server: values.server } : {}),
-            ...(this.jurisdiction_selected_shortcode ? { company: this.jurisdiction_selected_shortcode } : {}),
+            ...(this.jurisdiction_selected_shortcode
+                ? { company: this.account_type.category === 'real' ? this.jurisdiction_selected_shortcode : 'svg' }
```

Comment:
We need to keep the company empty for demo accounts, let's not use svg.

Reply:
Getting this error when it's empty

<img width="682" alt="Screenshot 2024-06-12 at 12 48 34 PM" src="https://github.com/binary-com/deriv-app/assets/126637868/a8e049a8-8dbc-4aad-a978-4d9172af5d1d">